caluma/form/serializers.py

```diff
@@ -534,12 +534,16 @@ class SaveDocumentTableAnswerSerializer(SaveAnswerSerializer):
     )
 
     def _get_document_tree(self, document_id):
-        answers = models.AnswerDocument.objects.filter(document_id=document_id).values(
-            "answer"
+        """Get all child documents (table rows) of a given document, and the document itself."""
+
+        answers = models.Answer.objects.filter(document_id=document_id)
```

Comment:
better to be filter after type `TYPE_TABLE` as well? Could then call it table_answers to make the code better understood.